url.bs

```diff
@@ -1068,6 +1068,33 @@ code point is U+003A (:).
 <p class="note">As per the <a href=#url-writing>URL writing</a> section, only a
 <a>normalized Windows drive letter</a> is conforming.
 
+<p>A string <dfn>starts with Windows drive letter</dfn> if all of the following are true:
+
+<ul>
+ <li><p>it contains at least two code points and the first two code points are
+ <a>Windows drive letter</a>.
```

Comment:
its length is greater than 1*

I'd give the second requirement its own bullet point since we already AND the whole list

Reply:
Also, no bullet at the end here since the sentence isn't finished

Reply:
a Windows drive letter*